tests/framework/utils_vsock.py

```diff
@@ -83,6 +85,8 @@ def exit(self):
         This method can be called from any thread. Upon returning, the
         echo server will have shut down.
         """
+        self.sock.close()
+        utils.run_cmd("rm -f {}".format(self.path))
```

Comment:
nit: I think you could use `sock.getsockname()` to avoid storing an extra variable